Index: kfilereplace/kfilereplacepart.desktop
===================================================================
--- kfilereplace/kfilereplacepart.desktop	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ kfilereplace/kfilereplacepart.desktop	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -24,6 +24,7 @@
 Comment[is]=Tól til að leita að texta og skipta út
 Comment[it]=Uno strumento per la ricerca e la sostituzione in sequenza
 Comment[ja]=バッチ検索及び置換ツール
+Comment[lt]=Paketinis paieškos ir keitimo įrankis
 Comment[ms]=Gugusan alatan cari dan ganti
 Comment[nl]=Een gereedschap voor zoeken en vervangen
 Comment[pl]=Narzędzie do wyszukiwania i zamiany
@@ -38,7 +39,6 @@
 Comment[tg]=Ҷустуҷӯи пакет ва ивази асбоб
 Comment[tr]=Bir toplu arama ve değiştirme aracı
 Comment[uk]=Засіб пакетного пошуку та заміни
-Comment[xx]=xxA batch search and replace toolxx
 Comment[zh_CN]=批量搜索替换工具
 Name=KFileReplace
 Name[hi]=के-फ़ाइल-रिप्लेस
@@ -46,7 +46,6 @@
 Name[ta]=Kகோப்பு இடமாற்று
 Name[tg]=KФайлро ҷойгузор кардан
 Name[tr]=K Dosya Değiştirme
-Name[xx]=xxKFileReplacexx
 GenericName=File Replace View
 GenericName[bg]=Преглед на файловете за замяна
 GenericName[ca]=Vista per a substituir fitxers
@@ -63,6 +62,7 @@
 GenericName[is]=Texta útskiptingarsýn
 GenericName[it]=Vista di sostituzione file
 GenericName[ja]=ファイル置換ビュー
+GenericName[lt]=Keitimų bylose vaizdas
 GenericName[ms]=Paparan Ganti Fail
 GenericName[nl]=Weergave van bestanden vervangen
 GenericName[pl]=Widok zamiany w plikach
@@ -75,5 +75,4 @@
 GenericName[sv]=Filersättningsvy
 GenericName[ta]=கோப்பு இடமாற்று காட்சி
 GenericName[uk]=Перегляд заміни в файлах
-GenericName[xx]=xxFile Replace Viewxx
 GenericName[zh_CN]=文件替换视图
Index: kfilereplace/kfilereplace.desktop
===================================================================
--- kfilereplace/kfilereplace.desktop	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ kfilereplace/kfilereplace.desktop	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -6,7 +6,6 @@
 Name[ta]=Kகோப்பு இடமாற்று
 Name[tg]=KФайлро ҷойгузор кардан
 Name[tr]=K Dosya Değiştirme
-Name[xx]=xxKFileReplacexx
 Exec=kfilereplace %i %m -caption "%c"
 Icon=kfilereplace
 Type=Application
@@ -29,6 +28,7 @@
 GenericName[is]=Tól til að leita að texta og skipta út
 GenericName[it]=Strumento di ricerca e sostituzione
 GenericName[ja]=検索及び置換ツール
+GenericName[lt]=Paieškos ir keitimo įrankis
 GenericName[nl]=Gereedschap voor zoeken en vervangen
 GenericName[pl]=Narzędzie do wyszukiwania i zamiany
 GenericName[pt]=Ferramenta de Procura e Substituição
@@ -39,6 +39,5 @@
 GenericName[sr@Latn]=Alat za pretragu i zamenu
 GenericName[sv]=Sök- och ersättningsverktyg
 GenericName[uk]=Засіб пошуку та заміни
-GenericName[xx]=xxSearch & Replace Toolxx
 GenericName[zh_CN]=搜索替换工具
 Categories=Qt;KDE;Utility;X-KDE-Utilities-File;
Index: quanta/parsers/parser.h
===================================================================
--- quanta/parsers/parser.h	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/parsers/parser.h	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -83,9 +83,6 @@
    * Crash errors of Parser::nodeAt is a good sign of a missing setRootNode
    */
   void setRootNode(Node* node) {m_node = node;} //TODO: check if m_saParser should be updated or not!
-  /** Print the doc structure tree to the standard output.
-      Only for debugging purposes. */
-  void coutTree(Node *node, int indent);
   void synchParseInDetail();
 
   IncludedGroupElementsMap includedMap;
Index: quanta/parsers/parsercommon.cpp
===================================================================
--- quanta/parsers/parsercommon.cpp	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/parsers/parsercommon.cpp	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -15,10 +15,11 @@
  ***************************************************************************/
 
 //qt includes
-#include "qstring.h"
+#include <qstring.h>
 
 //kde includes
-#include "klocale.h"
+#include <kdebug.h>
+#include <klocale.h>
 #include <ktexteditor/editinterface.h>
 
 //own includes
@@ -202,4 +203,37 @@
   return node;
 }
 
+void coutTree(Node *node, int indent)
+{
+    QString output;
+    int bLine, bCol, eLine, eCol;
+    if (!node)
+        kdDebug(24000)<< "undoRedo::coutTree() - bad node!" << endl;
+    while (node)
+    {
+            output = "";
+            output.fill('.', indent);
+            node->tag->beginPos(bLine, bCol);
+            node->tag->endPos(eLine, eCol);
+            if (node->tag->type != Tag::Text)
+                    output += node->tag->name.replace('\n'," ");
+            else
+                    output+= node->tag->tagStr().replace('\n'," ");
+            kdDebug(24000) << output <<" (" << node->tag->type << ") at pos " <<
+                    bLine << ":" << bCol << " - " << eLine << ":" << eCol << " This: "<< node << " Parent: " << node->parent << " Prev: " << node->prev << " Next: " << node->next << " Child: " << node->child << endl;
+ /*           for(j = 0; j < node->tag->attrCount(); j++)
+            {
+                    kdDebug(24000)<< " attr" << j << " " <<
+                            node->tag->getAttribute(j).nameLine << ":" <<
+                            node->tag->getAttribute(j).nameCol << " - " <<
+                            node->tag->getAttribute(j).valueLine << ":" <<
+                            node->tag->getAttribute(j).valueCol << endl;
+            }
+*/
+            if (node->child)
+                    coutTree(node->child, indent + 4);
+            node = node->next;
+    }
 }
+
+}
Index: quanta/parsers/parsercommon.h
===================================================================
--- quanta/parsers/parsercommon.h	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/parsers/parsercommon.h	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -49,6 +49,10 @@
   */
   Node* createScriptTagNode(Document *write, const AreaStruct &area, const QString &areaName,
                             const DTDStruct *dtd, Node *parentNode, Node *currentNode);
+
+/** Print the doc structure tree to the standard output.
+    Only for debugging purposes. */
+  void coutTree(Node *node, int indent);
 }
  
 #endif
Index: quanta/parsers/saparser.cpp
===================================================================
--- quanta/parsers/saparser.cpp	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/parsers/saparser.cpp	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -900,7 +900,11 @@
         if (m_currentNode->next->closesPrevious)
         {
           m_currentNode->next->removeAll = false;
+          Node *secondNext = m_currentNode->next->next;
+          if (secondNext)
+            secondNext->prev = m_currentNode;
           delete m_currentNode->next;
+          m_currentNode->next = secondNext;
           m_useNext = true;
         }
       } else
Index: quanta/parsers/parser.cpp
===================================================================
--- quanta/parsers/parser.cpp	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/parsers/parser.cpp	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -535,12 +535,12 @@
     m_node = ParserCommon::createTextNode(w, 0L, maxLines, w->editIf->lineLength(maxLines), 0L);
   }
   m_parsingNeeded = false;
-/*
- treeSize = 0;
- kdDebug(24000) << "Basenode : " << m_node << endl;
- coutTree(m_node, 2);
- kdDebug(24000) << "Size of tree: " << treeSize << endl;
-*/
+
+//  treeSize = 0;
+//  kdDebug(24000) << "Basenode : " << m_node << endl;
+//  ParserCommon::coutTree(m_node, 2);
+//  kdDebug(24000) << "Size of tree: " << treeSize << endl;
+
 //FIXME: What is the use of two pointer to the same Node???
  baseNode = m_node;
  m_saParser->init(m_node, w);
@@ -556,41 +556,7 @@
  return m_node;
 }
 
-/** Print the doc structure tree to the standard output.
-    Only for debugging purposes. */
 
-void Parser::coutTree(Node *node, int indent)
-{
-    QString output;
-    int bLine, bCol, eLine, eCol;
-    if (!node)
-        kdDebug(24000)<< "undoRedo::coutTree() - bad node!" << endl;
-    while (node)
-    {
-            output = "";
-            output.fill('.', indent);
-            node->tag->beginPos(bLine, bCol);
-            node->tag->endPos(eLine, eCol);
-            if (node->tag->type != Tag::Text)
-                    output += node->tag->name.replace('\n'," ");
-            else
-                    output+= node->tag->tagStr().replace('\n'," ");
-            kdDebug(24000) << output <<" (" << node->tag->type << ") at pos " <<
-                    bLine << ":" << bCol << " - " << eLine << ":" << eCol << " This: "<< node << " Parent: " << node->parent << " Prev: " << node->prev << " Next: " << node->next << " Child: " << node->child << endl;
- /*           for(j = 0; j < node->tag->attrCount(); j++)
-            {
-                    kdDebug(24000)<< " attr" << j << " " <<
-                            node->tag->getAttribute(j).nameLine << ":" <<
-                            node->tag->getAttribute(j).nameCol << " - " <<
-                            node->tag->getAttribute(j).valueLine << ":" <<
-                            node->tag->getAttribute(j).valueCol << endl;
-            }
-*/
-            if (node->child)
-                    coutTree(node->child, indent + 4);
-            node = node->next;
-    }
-}
 
 
 /** No descriptions */
@@ -976,9 +942,9 @@
     node = nextNode;
 
  //   kdDebug(24000)<< "Node removed!" << endl;
-//    coutTree(m_node, 2);
+//    ParserCommon::coutTree(m_node, 2);
   }
-//  coutTree(m_node, 2);
+//  ParserCommon::coutTree(m_node, 2);
 }
 
 Node *Parser::rebuild(Document *w)
@@ -1001,7 +967,7 @@
 
  //**kdDebug(24000)<< "************* Begin User Modification *****************" << endl;
   //debug!
-  //coutTree(m_node, 2);//*/
+  //ParserCommon::coutTree(m_node, 2);//*/
  if (w != write || !m_node) //the document has changed or the top node does not exists => parse completely
  {
   logReparse(modifs, w);
@@ -1217,17 +1183,16 @@
    }
  }
 /*   kdDebug(24000)<< "END"<< endl;
-   coutTree(baseNode,  2);
+   ParserCommon::coutTree(baseNode,  2);
    kdDebug(24000)<< "************* End User Modification *****************" << endl;*/
    w->docUndoRedo->addNewModifsSet(modifs, undoRedo::SourceModif);
  }
   kdDebug(24000) << "Rebuild: " << t.elapsed() << " ms \n";
-/*
- treeSize = 0;
- coutTree(m_node, 2);
- kdDebug(24000) << "Size of tree: " << treeSize << endl;
-// cout << "\n"; */
 
+/* treeSize = 0;
+ ParserCommon::coutTree(m_node, 2);
+ kdDebug(24000) << "Size of tree: " << treeSize << endl;*/
+
  m_saParser->init(m_node, w);
  if (saParserEnabled)
    QTimer::singleShot(0, this, SLOT(slotParseInDetail()));
Index: quanta/dialogs/dtepeditdlgs.ui
===================================================================
--- quanta/dialogs/dtepeditdlgs.ui	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/dialogs/dtepeditdlgs.ui	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -606,7 +606,7 @@
                             <string>Enable the second extra page</string>
                         </property>
                         <property name="toolTip" stdset="0">
-                            <string>See the tooltip and what's this information for the first checkbox</string>
+                            <string>See the tooltip and "What's This?" information for the first checkbox</string>
                         </property>
                     </widget>
                     <widget class="QCheckBox" row="6" column="0" rowspan="1" colspan="3">
@@ -620,7 +620,7 @@
                             <string>Enable the third extra page</string>
                         </property>
                         <property name="toolTip" stdset="0">
-                            <string>See the tooltip and what's this information for the first checkbox</string>
+                            <string>See the tooltip and "What's This?" information for the first checkbox</string>
                         </property>
                     </widget>
                     <widget class="QCheckBox" row="9" column="0" rowspan="1" colspan="3">
@@ -634,7 +634,7 @@
                             <string>Enable the fourth extra page</string>
                         </property>
                         <property name="toolTip" stdset="0">
-                            <string>See the tooltip and what's this information for the first checkbox</string>
+                            <string>See the tooltip and "What's This?" information for the first checkbox</string>
                         </property>
                     </widget>
                     <widget class="QCheckBox" row="12" column="0" rowspan="1" colspan="3">
@@ -648,7 +648,7 @@
                             <string>Enable the fifth extra page</string>
                         </property>
                         <property name="toolTip" stdset="0">
-                            <string>See the tooltip and what's this information for the first checkbox</string>
+                            <string>See the tooltip and "What's This?" information for the first checkbox</string>
                         </property>
                     </widget>
                     <widget class="KLineEdit" row="1" column="2">
@@ -692,7 +692,7 @@
                             <bool>false</bool>
                         </property>
                         <property name="toolTip" stdset="0">
-                            <string>See the tooltip and what's this information for the first page fields</string>
+                            <string>See the tooltip and "What's This?" information for the first page fields</string>
                         </property>
                     </widget>
                     <widget class="QLabel" row="8" column="0">
@@ -706,7 +706,7 @@
                             <string>Groups:</string>
                         </property>
                         <property name="toolTip" stdset="0">
-                            <string>See the tooltip and what's this information for the first page fields</string>
+                            <string>See the tooltip and "What's This?" information for the first page fields</string>
                         </property>
                     </widget>
                     <widget class="KLineEdit" row="4" column="2">
@@ -717,7 +717,7 @@
                             <bool>false</bool>
                         </property>
                         <property name="toolTip" stdset="0">
-                            <string>See the tooltip and what's this information for the first page fields</string>
+                            <string>See the tooltip and "What's This?" information for the first page fields</string>
                         </property>
                     </widget>
                     <widget class="QLabel" row="10" column="0" rowspan="1" colspan="2">
@@ -731,7 +731,7 @@
                             <string>Title:</string>
                         </property>
                         <property name="toolTip" stdset="0">
-                            <string>See the tooltip and what's this information for the first page fields</string>
+                            <string>See the tooltip and "What's This?" information for the first page fields</string>
                         </property>
                     </widget>
                     <widget class="QLabel" row="4" column="0">
@@ -745,7 +745,7 @@
                             <string>Title:</string>
                         </property>
                         <property name="toolTip" stdset="0">
-                            <string>See the tooltip and what's this information for the first page fields</string>
+                            <string>See the tooltip and "What's This?" information for the first page fields</string>
                         </property>
                     </widget>
                     <widget class="QLabel" row="5" column="0">
@@ -759,7 +759,7 @@
                             <string>Groups:</string>
                         </property>
                         <property name="toolTip" stdset="0">
-                            <string>See the tooltip and what's this information for the first page fields</string>
+                            <string>See the tooltip and "What's This?" information for the first page fields</string>
                         </property>
                     </widget>
                     <widget class="QLabel" row="7" column="0">
@@ -773,7 +773,7 @@
                             <string>Title:</string>
                         </property>
                         <property name="toolTip" stdset="0">
-                            <string>See the tooltip and what's this information for the first page fields</string>
+                            <string>See the tooltip and "What's This?" information for the first page fields</string>
                         </property>
                     </widget>
                     <widget class="QLabel" row="11" column="0" rowspan="1" colspan="2">
@@ -787,7 +787,7 @@
                             <string>Groups:</string>
                         </property>
                         <property name="toolTip" stdset="0">
-                            <string>See the tooltip and what's this information for the first page fields</string>
+                            <string>See the tooltip and "What's This?" information for the first page fields</string>
                         </property>
                     </widget>
                     <widget class="KLineEdit" row="5" column="2">
@@ -798,7 +798,7 @@
                             <bool>false</bool>
                         </property>
                         <property name="toolTip" stdset="0">
-                            <string>See the tooltip and what's this information for the first page fields</string>
+                            <string>See the tooltip and "What's This?" information for the first page fields</string>
                         </property>
                     </widget>
                     <widget class="QLabel" row="13" column="0">
@@ -812,7 +812,7 @@
                             <string>Title:</string>
                         </property>
                         <property name="toolTip" stdset="0">
-                            <string>See the tooltip and what's this information for the first page fields</string>
+                            <string>See the tooltip and "What's This?" information for the first page fields</string>
                         </property>
                     </widget>
                     <widget class="KLineEdit" row="11" column="2">
@@ -823,7 +823,7 @@
                             <bool>false</bool>
                         </property>
                         <property name="toolTip" stdset="0">
-                            <string>See the tooltip and what's this information for the first page fields</string>
+                            <string>See the tooltip and "What's This?" information for the first page fields</string>
                         </property>
                     </widget>
                     <widget class="QLabel" row="14" column="0">
@@ -837,7 +837,7 @@
                             <string>Groups:</string>
                         </property>
                         <property name="toolTip" stdset="0">
-                            <string>See the tooltip and what's this information for the first page fields</string>
+                            <string>See the tooltip and "What's This?" information for the first page fields</string>
                         </property>
                     </widget>
                     <widget class="KLineEdit" row="7" column="1" rowspan="1" colspan="2">
@@ -848,7 +848,7 @@
                             <bool>false</bool>
                         </property>
                         <property name="toolTip" stdset="0">
-                            <string>See the tooltip and what's this information for the first page fields</string>
+                            <string>See the tooltip and "What's This?" information for the first page fields</string>
                         </property>
                     </widget>
                     <widget class="KLineEdit" row="8" column="1" rowspan="1" colspan="2">
@@ -859,7 +859,7 @@
                             <bool>false</bool>
                         </property>
                         <property name="toolTip" stdset="0">
-                            <string>See the tooltip and what's this information for the first page fields</string>
+                            <string>See the tooltip and "What's This?" information for the first page fields</string>
                         </property>
                     </widget>
                     <widget class="KLineEdit" row="14" column="2">
@@ -870,7 +870,7 @@
                             <bool>false</bool>
                         </property>
                         <property name="toolTip" stdset="0">
-                            <string>See the tooltip and what's this information for the first page fields</string>
+                            <string>See the tooltip and "What's This?" information for the first page fields</string>
                         </property>
                     </widget>
                     <widget class="KLineEdit" row="13" column="2">
@@ -881,7 +881,7 @@
                             <bool>false</bool>
                         </property>
                         <property name="toolTip" stdset="0">
-                            <string>See the tooltip and what's this information for the first page fields</string>
+                            <string>See the tooltip and "What's This?" information for the first page fields</string>
                         </property>
                     </widget>
                 </grid>
Index: quanta/dialogs/dtepeditdlg.cpp
===================================================================
--- quanta/dialogs/dtepeditdlg.cpp	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/dialogs/dtepeditdlg.cpp	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -58,11 +58,12 @@
 
 void DTEPEditDlg::slotFamilyChanged(int family)
 {
-  m_family = family;
-  if (m_family == 0)
+  if (family == 0 && m_family != 0)
     tabWidget->insertTab(m_pagesWidget, i18n("&Pages"), 1);
   else
+  if (family == 1)
    tabWidget->removePage(m_pagesWidget);
+  m_family = family;
 }
 
 void DTEPEditDlg::init()
Index: quanta/components/debugger/variableslistview.cpp
===================================================================
--- quanta/components/debugger/variableslistview.cpp	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/components/debugger/variableslistview.cpp	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -215,7 +215,7 @@
   item->setText(VariablesListViewColumns::Name, oldvar->name());
   item->setText(VariablesListViewColumns::Type, oldvar->typeName());
   item->setText(VariablesListViewColumns::Size, oldvar->sizeName());
-  item->setText(VariablesListViewColumns::Value, (newvar->isScalar() ? oldvar->value() : ""));
+  item->setText(VariablesListViewColumns::Value, (newvar->isScalar() ? oldvar->value() : QString()));
   
 }
 
Index: quanta/components/debugger/interfaces/quantadebugger.desktop
===================================================================
--- quanta/components/debugger/interfaces/quantadebugger.desktop	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/components/debugger/interfaces/quantadebugger.desktop	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -10,7 +10,7 @@
 Comment[el]=Πρόσθεμα αποσφαλματωτή Quanta 
 Comment[es]=Un accesorio para un depurador de Quanta
 Comment[et]=Quanta siluriplugin
-Comment[eu]=Quanta araztatzailearen plugina
+Comment[eu]=Quanta araztailearen plugina
 Comment[fi]=Quantan debuggerin plugin
 Comment[fr]=Un module de débogage pour Quanta
 Comment[ga]=Breiseán dífhabhtóra Quanta
@@ -18,6 +18,7 @@
 Comment[is]=Quanta aflúsunaríforrit
 Comment[it]=Un plugin debugger di Quanta
 Comment[ja]=Quanta デバッガプラグイン
+Comment[lt]=Quanta derintuvės priedas
 Comment[ms]=Plug masuk penyah-ralat Quanta
 Comment[nl]=Een Quanta debugger-plugin
 Comment[pl]=Wtyczka debuggera Quanty
@@ -32,5 +33,4 @@
 Comment[tg]=Модули ғалатёби Quanta
 Comment[tr]=Bir Quanta Hata Ayıklayıcı eklentisi
 Comment[uk]=Втулок зневадження Quanta
-Comment[xx]=xxA Quanta Debugger pluginxx
 Comment[zh_CN]=Quanta 调试器插件
Index: quanta/components/debugger/gubed/gubedsettingss.ui
===================================================================
--- quanta/components/debugger/gubed/gubedsettingss.ui	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/components/debugger/gubed/gubedsettingss.ui	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -326,7 +326,7 @@
                                     <cstring>lineStartSession</cstring>
                                 </property>
                                 <property name="toolTip" stdset="0">
-                                    <string>See 'Whats this" for available variables</string>
+                                    <string>See "What's This?" for available variables</string>
                                 </property>
                                 <property name="whatsThis" stdset="0">
                                     <string>%afn - Filename of the current script
Index: quanta/components/debugger/gubed/quantadebuggergubed.cpp
===================================================================
--- quanta/components/debugger/gubed/quantadebuggergubed.cpp	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/components/debugger/gubed/quantadebuggergubed.cpp	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -43,7 +43,7 @@
 
 
 K_EXPORT_COMPONENT_FACTORY( quantadebuggergubed,
-                            KGenericFactory<QuantaDebuggerGubed>("quantadebuggergubed"));
+                            KGenericFactory<QuantaDebuggerGubed>("quantadebuggergubed"))
 
 const char QuantaDebuggerGubed::protocolversion[] = "0.0.12";
 
Index: quanta/components/debugger/gubed/quantadebuggergubed.desktop
===================================================================
--- quanta/components/debugger/gubed/quantadebuggergubed.desktop	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/components/debugger/gubed/quantadebuggergubed.desktop	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -3,7 +3,6 @@
 Encoding=UTF-8
 Name=Gubed
 Name[ta]=குபெட்
-Name[xx]=xxGubedxx
 Comment=Quanta debugger plugin to interact with the Gubed PHP debugger, see http://gubed.sf.net
 Comment[ca]=Connector de depuració pel Quanta que interactua amb el depurador Gubed PHP, consulteu http://gubed.sf.net
 Comment[cs]=Ladicí modul Quanty s Gubed PHP debuggerem, viz http://gubed.sf.net
@@ -12,13 +11,14 @@
 Comment[el]=Αποσφαλματωτής Quanta σε συνεργασία με το αποσφαλματωτή Gubed PHP, δείτε http://gubed.sf.net
 Comment[es]=Accesorio de depuración de Quanta, para interactuar con el depurador de PHP Gubed. Vea http://gubed.sf.net
 Comment[et]=Quanta siluriplugin koostööks PHP siluriga Gubed (vaata http://gubed.sf.net)
-Comment[eu]=Gubed PHP araztatzailearekin lan egiteko Quanta araztatzailearen plugina, ikusi http://gubed.sf.net
+Comment[eu]=Gubed PHP araztailearekin lan egiteko Quanta araztailearen plugina, ikusi http://gubed.sf.net
 Comment[fi]=Gubed PHP debuggerin plugin Quantan debuggeriin (katso http://gubed.sf.net)
 Comment[fr]=Module de débogage de Quanta pour interagir avec le débogueur PHP Gubed ; consulter http://gubed.sf.net.
 Comment[hu]=Quanta-nyomkövető (bővítőmodulként) a Gubed PHP-nyomkövetőhöz, lásd: http://gubed.sf.net
 Comment[is]=Quanta aflúsunaríforrit til samskipta við Gubed PHP aflúsarann. Sjá http://gubed.sf.net
 Comment[it]=Plugin debugger di Quanta per interagire con il debugger PHP Gubed, vedi http://gubed.sf.net
 Comment[ja]=Gubed PHP デバッガと対話する Quanta デバッガ。詳細は http://gubed.sf.net
+Comment[lt]=Quanta derintuvės priedas skirtas dirbti su Gubed PHP derintuve, žr.http://gubed.sf.net
 Comment[ms]=Plug masuk penyah-ralat Quanta untuk berinteraksi dengan penyah-ralat Gubed PHP, lihat http://gubed.sf.net
 Comment[nl]=Quanta debugger plugin om te communiceren met de Gubed PHP debugger, zie http://gubed.sf.net
 Comment[pl]=Wtyczka debuggera Quanty współpracująca z debuggerem PHP Gubed , patrz http://gubed.sf.net
@@ -33,7 +33,6 @@
 Comment[tg]=Модули ғалатёби Quanta барои алоқаи аз Gubed, ғалатёби PHP, нишон кунед http://gubed.sf.net
 Comment[tr]=Gubed PHP hata ayıklayıcısıyla iletişim kurmayı sağlayan Quanta hata ayıklama eklentisi, http://gubed.sf.net adresini ziyaret edin
 Comment[uk]=Втулок зневадження Quanta для зв'язку зі зневаджувачем PHP -- Gubed, див. http://gubed.sf.net
-Comment[xx]=xxQuanta debugger plugin to interact with the Gubed PHP debugger, see http://gubed.sf.netxx
 Comment[zh_CN]=与 Gubed PHP 调试器交互 Quanta 调试器插件，见 http://gubed.sf.net
 Icon=kdbg
 ServiceTypes=Quanta/Debugger
Index: quanta/components/debugger/dbgp/quantadebuggerdbgp.desktop
===================================================================
--- quanta/components/debugger/dbgp/quantadebuggerdbgp.desktop	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/components/debugger/dbgp/quantadebuggerdbgp.desktop	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -2,7 +2,6 @@
 Type=Service
 Encoding=UTF-8
 Name=DBGp
-Name[xx]=xxDBGpxx
 Comment=DBGp debugger plugin, see http://xdebug.org
 Comment[ca]=Connector de depuració DBGp, consulteu http://xdebug.org
 Comment[cs]=DBGp debugger plugin, viz http://xdebug.org
@@ -11,20 +10,24 @@
 Comment[el]=Πρόσθετο αποσφαλματωτή DBGp, δείτε το http://xdebug.org
 Comment[es]=Extensión para el depurador DBGp, vea http://xdebug.org
 Comment[et]=DBGp siluri plugin, vaata http://xdebug.org
+Comment[eu]=DBGp araztailearen plugina, ikusi http://xdebug.org
 Comment[fi]=DBGp-debugliitännäinen, katso http://xdebug.org
 Comment[fr]=Module de débogage DBGp, consultez http://xdebug.org
 Comment[ga]=Breiseán dífhabhtóra DBGp, féach ar http://xdebug.org
+Comment[hu]=DBGp nyomkövető modul, lásd: http://xdebug.org
+Comment[is]=DBGp aflúsunaríforrit. Sjá http://xdebug.org
 Comment[it]=Plugin di degub DBGp, vedi http://xdebug.org
+Comment[lt]=DBGp derintuvės priedas, žr. http://xdebug.org
 Comment[nl]=DBGp-debugger-plugin, zie http://xdebug.org
 Comment[pl]=Wtyczka debugera DBGp, patrz http://xdebug.org
 Comment[pt]='Plugin' de depuração DBGp, veja http://xdebug.org
 Comment[pt_BR]='Plugin' de depuração DBGp, veja http://xdebug.org
 Comment[ru]=Модуль для  отладчика DBGp, http://xdebug.org
+Comment[sl]=Vstavek za razhoroščevanje DBGp, glej http://xdebug.org
 Comment[sr]=DBGp прикључак, види http://xdebug.org
 Comment[sr@Latn]=DBGp priključak, vidi http://xdebug.org
 Comment[sv]=DBGp-insticksprogram för felsökning, se http://xdebug.org
 Comment[uk]=Втулок зневаджувача DBGp, див. http://xdebug.org
-Comment[xx]=xxDBGp debugger plugin, see http://xdebug.orgxx
 Comment[zh_CN]=DBGp 调试器插件，参看 http://xdebug.org
 Icon=kdbg
 ServiceTypes=Quanta/Debugger
Index: quanta/components/debugger/dbgp/dbgpsettingss.ui
===================================================================
--- quanta/components/debugger/dbgp/dbgpsettingss.ui	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/components/debugger/dbgp/dbgpsettingss.ui	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -185,7 +185,7 @@
                                     </sizepolicy>
                                 </property>
                                 <property name="toolTip" stdset="0">
-                                    <string>See 'Whats this" for available variables</string>
+                                    <string>See "What's This?" for available variables</string>
                                 </property>
                                 <property name="whatsThis" stdset="0">
                                     <string>%afn - Filename of the current script
@@ -354,7 +354,7 @@
                                     </sizepolicy>
                                 </property>
                                 <property name="toolTip" stdset="0">
-                                    <string>See 'Whats this" for available variables</string>
+                                    <string>See "What's This?" for available variables</string>
                                 </property>
                                 <property name="whatsThis" stdset="0">
                                     <string>Path to the profiler output
Index: quanta/components/debugger/dbgp/quantadebuggerdbgp.cpp
===================================================================
--- quanta/components/debugger/dbgp/quantadebuggerdbgp.cpp	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/components/debugger/dbgp/quantadebuggerdbgp.cpp	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -910,7 +910,7 @@
 {
   if(attribute(setnode, "success") == "0")
   {
-    debuggerInterface()->showStatus(i18n("Unable to set value of variable!"), true);
+    debuggerInterface()->showStatus(i18n("Unable to set value of variable."), true);
   }
 }
 
Index: quanta/ChangeLog
===================================================================
--- quanta/ChangeLog	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/ChangeLog	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -1,38 +1,30 @@
-This files contains the changes since Quanta 2.0 until the current 3.4.x series.
+This files contains the changes since Quanta 2.0 until the current 3.5.x series.
 
-Version 3.5 (Release date: xx-xx-2004; Started 04-03-2004):
+Version 3.5.1 (Release date: 29-11-2005; Started 04-03-2004):
  - bugfixes:
-        - read/write the correct entry from description.rc in the editor
-        - fill only the attributes with source="selection" with the selection in the
-          tag editing dialog
-        - fix matching of excluded files from a project
+        - better handling of quotation marks when editing tags inside a script area [#118693]
+        - don't show the file changed dialog after using save as and save again
+        - crash fixes in VPL [#118686]
+        - don't loose the comment closing character when formatting the XML code [#118453]
+        - insert valid img tag for XHTML documents [#118805]
+        - don't show the Pages tab in DTEP editing dialog more than once [#118840]
+        - set the DTEP of the document to the one selected in the Quick Start dialog [#118814]
+        - don't have two Close actions [#118448]
+
+ - improvements:
+        - add XHTML 1.1 and XHTML 1.0 Basic to the quickstart dialog [#118813]
+        - new DCOP interfaces/methods:
+            - WindowManagerIf::setDtep
+
+
+Version 3.5 (Release date: 29-11-2005; Started 04-03-2004):
+ - bugfixes:
         - don't crash when Smart Tag Insertion is enabled and some tags are inserted to an
           emty document
-        - don't crash when Selected is pressed and nothing is selected (in the CSS editor) [#101919]
-        - show the correct relative paths in URL autocompletion
-        - fix crash when unsetting breakpoints
-        - entity autocompletion fixes
-        - show an upload treeview for profiles that have @ in the username
-        - allow CVS actions to be put on toolbars [#106023]
-        - open the files with the right encoding when restoring the last opened files on startup [#104512]
         - add missing Romanian characters to the Insert Special Character dialog [#107549]
-        - treat "<" as the end of a tag name. Fixes autoreplacement of opening/closing
-          tags in cases when a PHP area starts directly after a tag name. [#106911]
-        - fix the Kommander XML scripts.
-        - fix crash when deleting a file which is part of a project from the tab
-          context menu [#111134]
         - fix layout of mailto dialogs
-        - do not autoclose tag starting with <?, like <?xml
-        - create empty description files for scripts that don't have one
-        - fix loading order of the project view files
-        - save the content entered in a new file in the VPL editor [#111278]
-        - do not allow invocation of the CSS editor in an empty non-CSS document [#109815]
-        - fix many cell-merging related errors in the table editor [#112243]
-        - fix namespace editing in the attribute editor tree
-        - replace a leading ~ in an upload profile with the users home folder and avoid a hang
         - fix lots of VPL related crashes [#112733, #112853, #108501, #115046]
         - give to the host field more space in the upload profile dialog [#112778]
-        - show DT tags in VPL [#109723]
         - remember find options between sessions [#97102]
         - don't hide the password dialogs behind the main window [#114584]
         - replace < and > typed in VPL with &lt; and &gt; in source [#115528]
@@ -40,19 +32,12 @@
 
  - improvements:
         - md5sum based file changed detection
-        - nicer processing of CVS output
-        - read the image sizes for remote images as well
-        - insert an img tag for remote images as well when using D&D
-        - better context sensitiv documentation (needs updated documentation and DTEP
-          packages)
         - faster loading of multiple documents at once
         - possibility to change the icon/text layout of the user toolbars from
           inside Quanta added
         - external preview possibility with Firefox added [#106557]
         - make autoreplacing of accented chars smarter. The characters are replaced only if they cannot
       	  be encoded in the encoding of the document
-        - make single file/folder upload really quick [#105612]
-        - Paste as HTML Encoded encodes more chars to their corresponding entities [#100547, #100103]
         - escape the quotation marks of the inserted attributes inside a script
           area if needed
         - less annoying CSS autocompletion [#109603]
@@ -66,8 +51,58 @@
  - new features:
         - loading/updating of the entities in a DTEP from a DTD
 
+Version 3.4.3 (Release date: 05-10-2005; Started 01-08-2005):
+ - bugfixes:
+        - fix crash when deleting a file which is part of a project from the tab
+          context menu [#111134]
+        - do not autoclose tag starting with <?, like <?xml
+        - create empty description files for scripts that don't have one
+        - do not allow invocation of the CSS editor in an empty non-CSS document [#109815]
+        - fix many cell-merging related errors in the table editor [#112243]
+        - fix namespace editing in the attribute editor tree
+        - fix lots of table editor bugs
+        - restore automatic conversion of accented chars behavior to pre-3.4.2
+        - save the content entered in a new file in the VPL editor [#111278]
+        - fix loading order of the project view files
+        - replace a leading ~ in an upload profile with the users home folder and avoid a hang
+  - fix lots of VPL related crashes
+  - show DT tags in VPL [#109723]
+  
+Version 3.4.2 (Release date: 20-07-2005; Started 01-06-2005):
+ - bugfixes:
+        - entity autocompletion fixes
+        - don't show the name twice in the tooltip of treeview items
+        - show an upload treeview for profiles that have @ in the username
+        - allow CVS actions to be put on toolbars [#106023]
+        - open the files with the right encoding when restoring the last opened files on startup [#104512]
+        - fix the Kommander XML scripts
+        - treat "<" as the end of a tag name. Fixes autoreplacement of opening/closing
+                tags in cases when a PHP area starts directly after a tag name. [#106911]
 
-Version 3.4 (Release date: xx-03-2005; Started 20-08-2004):
+ - improvements:
+        - make single file/folder upload really quick [#105612]
+        - Paste as HTML Encoded encodes more chars to their corresponding entities [#100547, #100103]
+        - improve the automatic conversion of characters. Only those are converted that cannot be
+   encoded in the encoding of the document
+           
+Version 3.4.1 (Release date: 31-05-2005; Started 04-03-2004):
+ - bugfixes:
+        - read/write the correct entry from description.rc in the editor
+        - fill only the attributes with source="selection" with the selection in the
+          tag editing dialog
+        - fix matching of excluded files from a project
+        - don't crash when Selected is pressed and nothing is selected (in the CSS editor) [#101919]
+        - show the correct relative paths in URL autocompletion
+        - fix crash when unsetting breakpoints
+
+ - improvements:
+        - nicer processing of CVS output
+        - read the image sizes for remote images as well
+        - insert an img tag for remote images as well when using D&D
+        - better context sensitiv documentation (needs updated documentation and DTEP
+   packages)
+
+Version 3.4 (Release date: 16-03-2005; Started 20-08-2004):
   - bugfixes:
         - Debugger
           - Added full support for Gubeds conditional breakpoints
@@ -85,7 +120,7 @@
             a skeleton for VPL
           - fix "Cannot insert the tag: invalid location" problem [#99340]
           - restore the correct mouse cursor when the VPL area is left [#98177]
-	    - Other bug fixes.
+      - Other bug fixes.
         - show (again) the full filename in a tooltip
         - don't crash if the preview widget is closed with a JavaScript command
           from the code itself [#87533]
@@ -186,7 +221,7 @@
         - any action can be assigned to an event, not just the user defined
           script actions
         - more than one action can be assigned to events
-      	- possibility to disable the event actions added (without the need to
+        - possibility to disable the event actions added (without the need to
           remove them one by one)
         - support loading of more than one toolbar at a time
         - don't ask for toolbar saving if the toolbar names were modified by
@@ -233,8 +268,7 @@
         - PHP class autocompletion improved (autocomplete for $this; class methods; for classes from
           included files)
         - make CSS autocompletion less intrussive      
-      
-
+ 
 Version 3.3.0 (Release date: 18-08-2004; Started 07-05-2004):
 NOTE: Unfortunately I failed to update the changelog regulary, thus some new features and
 bugfixes might not be listed here.
Index: quanta/scripts/externalpreview.sh
===================================================================
--- quanta/scripts/externalpreview.sh	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/scripts/externalpreview.sh	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -15,7 +15,7 @@
 newTab=$3
 if test $currentURL
 then
-  if ! $browser -remote "openURL($currentURL, $newTab)"
+  if ! $browser -remote "openURL($currentURL,$newTab)"
   then
     $browser $currentURL
   fi
Index: quanta/scripts/htmlquickstart.kmdr
===================================================================
--- quanta/scripts/htmlquickstart.kmdr	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/scripts/htmlquickstart.kmdr	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -9,7 +9,7 @@
             <x>0</x>
             <y>0</y>
             <width>473</width>
-            <height>550</height>
+            <height>577</height>
         </rect>
     </property>
     <property name="sizePolicy">
@@ -160,7 +160,20 @@
 echo '@CBPHPFooter'
 echo '&lt;/body&gt;'
 echo '&lt;/html&gt;'
-</string>
+
+quanta=`dcop quanta-@parentPid`
+if [ -z "$quanta" ]; then
+    quanta=`dcop quanta`
+    if [ ! -z "$quanta" ]; then
+       quantadcop=quanta
+    fi
+else
+    quantadcop=quanta-@parentPid
+fi
+
+if [ ! -z "$quantadcop" ]; then
+  dcop $quantadcop WindowManagerIf setDtep "@DTEPselect.text" false
+fi</string>
                 </stringlist>
             </property>
             <grid>
@@ -237,7 +250,7 @@
                                 <cstring>CBPHPFooter</cstring>
                             </property>
                             <property name="text">
-                                <string>PHP footer include:</string>
+                                <string>&amp;PHP footer include:</string>
                             </property>
                             <property name="associations" stdset="0">
                                 <stringlist>
@@ -269,7 +282,7 @@
                         <cstring>CBDTD</cstring>
                     </property>
                     <property name="text">
-                        <string>Show DTD</string>
+                        <string>Show &amp;DTD</string>
                     </property>
                     <property name="checked">
                         <bool>true</bool>
@@ -314,6 +327,16 @@
                             <string>XHTML 1.0 Frameset</string>
                         </property>
                     </item>
+                    <item>
+                        <property name="text">
+                            <string>XHTML 1.0 Basic</string>
+                        </property>
+                    </item>
+                    <item>
+                        <property name="text">
+                            <string>XHTML 1.1</string>
+                        </property>
+                    </item>
                     <property name="name">
                         <cstring>DTEPselect</cstring>
                     </property>
@@ -340,6 +363,12 @@
 xhtml1frameset='&lt;?xml version="1.0" encoding="@ComboMetachar"?&gt;
 &lt;!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Frameset//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-frameset.dtd"&gt;'
 
+xhtml1basic='&lt;?xml version="1.0" encoding="@ComboMetachar"?&gt;
+&lt;!DOCTYPE html PUBLIC "-//W3C//DTD XHTML Basic 1.0//EN" "http://www.w3.org/TR/xhtml-basic/xhtml-basic10.dtd"&gt;'
+
+xhtml11='&lt;?xml version="1.0" encoding="@ComboMetachar"?&gt;
+&lt;!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd"&gt;'
+
 ################################
 # Set the DTD based on user selection
 
@@ -362,6 +391,12 @@
 "XHTML 1.0 Frameset" )
 	DTD="$xhtml1frameset"
 ;;
+"XHTML 1.0 Basic" )
+	DTD="$xhtml1basic"
+;;
+"XHTML 1.1" )
+	DTD="$xhtml11"
+;;
 * )
 	echo "No Match"
 ;;
@@ -375,7 +410,7 @@
                         <cstring>CBHead</cstring>
                     </property>
                     <property name="text">
-                        <string>Head area</string>
+                        <string>Head &amp;area</string>
                     </property>
                     <property name="checked">
                         <bool>true</bool>
@@ -403,7 +438,7 @@
                         <cstring>CBCVS</cstring>
                     </property>
                     <property name="text">
-                        <string>CVS tag in comment</string>
+                        <string>CVS tag &amp;in comment</string>
                     </property>
                     <property name="associations" stdset="0">
                         <stringlist>
@@ -443,7 +478,7 @@
                                 <cstring>CBMetaAuthor</cstring>
                             </property>
                             <property name="text">
-                                <string>Meta author:</string>
+                                <string>&amp;Meta author:</string>
                             </property>
                             <property name="associations" stdset="0">
                                 <stringlist>
@@ -473,7 +508,7 @@
                                 <cstring>CBStyleArea</cstring>
                             </property>
                             <property name="text">
-                                <string>Style area</string>
+                                <string>St&amp;yle area</string>
                             </property>
                             <property name="associations" stdset="0">
                                 <stringlist>
@@ -694,7 +729,7 @@
                                 <cstring>CBBasedir</cstring>
                             </property>
                             <property name="text">
-                                <string>Base directory:</string>
+                                <string>&amp;Base directory:</string>
                             </property>
                             <property name="associations" stdset="0">
                                 <stringlist>
@@ -709,7 +744,7 @@
                                 <cstring>CBMetaQuanta</cstring>
                             </property>
                             <property name="text">
-                                <string>Meta Quanta</string>
+                                <string>Meta &amp;Quanta</string>
                             </property>
                             <property name="checked">
                                 <bool>true</bool>
@@ -727,7 +762,7 @@
                                 <cstring>CBMetaKeywords</cstring>
                             </property>
                             <property name="text">
-                                <string>Meta keywords:</string>
+                                <string>Meta &amp;keywords:</string>
                             </property>
                             <property name="associations" stdset="0">
                                 <stringlist>
@@ -744,7 +779,7 @@
                                 <cstring>CBStyle</cstring>
                             </property>
                             <property name="text">
-                                <string>Linked style sheet:</string>
+                                <string>Li&amp;nked style sheet:</string>
                             </property>
                             <property name="associations" stdset="0">
                                 <stringlist>
@@ -793,7 +828,7 @@
                                 <cstring>CBJavascript</cstring>
                             </property>
                             <property name="text">
-                                <string>JavaScript area</string>
+                                <string>&amp;JavaScript area</string>
                             </property>
                             <property name="associations" stdset="0">
                                 <stringlist>
Index: quanta/src/quanta.h
===================================================================
--- quanta/src/quanta.h	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/src/quanta.h	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -151,6 +151,12 @@
   /** Return the list of opened URLs and their editor interface numbers*/
   QStringList openedURLs() const;
   QString saveCurrentFile();
+  /**
+   * Sets the DTEP for the current document.
+   * @param dtepName the name (nickname or full name) of the DTEP
+   * @param convert  if true, converts the !DOCTYPE line to the new DTEP
+   */
+  void setDtep(const QString& dtepName, bool convert);
   QStringList tagAreas(const QString& name, bool includeCoordinates, bool skipFoundContent) const;
   QString documentFolderForURL(const QString &url);
   QString urlWithPreviewPrefix(const QString &url);
@@ -247,7 +253,6 @@
   void slotFileSaveAll();
   void slotFileReload(QuantaView *view = 0L);
   void slotFileReloadAll();
-  void slotFileClose();
   /** Close the document specified in the parameter if it's opened */
   void slotFileClose(const KURL &url);
   void slotFileCloseAll();
Index: quanta/src/quanta_init.cpp
===================================================================
--- quanta/src/quanta_init.cpp	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/src/quanta_init.cpp	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -726,7 +726,6 @@
     //
     KStdAction::openNew( m_quanta, SLOT( slotFileNew()  ), ac);
     KStdAction::open   ( m_quanta, SLOT( slotFileOpen() ), ac, "file_open");
-    KStdAction::close  ( m_quanta, SLOT( slotFileClose()), ac);
     (void) new KAction(i18n("Close Other Tabs"), 0, ViewManager::ref(), SLOT(slotCloseOtherTabs()), ac, "close_other_tabs");
 
     m_quanta->fileRecent =  KStdAction::openRecent(m_quanta, SLOT(slotFileOpenRecent(const KURL&)),
Index: quanta/src/document.cpp
===================================================================
--- quanta/src/document.cpp	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/src/document.cpp	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -448,8 +448,23 @@
       if (s[i] == '"' && (i == 0 || s[i-1] != '\\'))
         insideQuotes = !insideQuotes;
     }
-    if (insideQuotes)
+    int eLine, eCol;
+    n->tag->endPos(eLine, eCol);
+    s = this->text(line + 1, col, eLine, eCol);
+    bool closeQuotationFound = false;
+    for (int i = 0 ; i < (int)s.length() - 1; i++)
+    {
+      if (s[i] == '"' && (i == 0 || s[i-1] != '\\'))
+      {
+        closeQuotationFound = true;
+        break;
+      }
+    }
+    if (insideQuotes && closeQuotationFound)
+    {
+      text.replace("\\\"", "\"");
       text.replace("\"", "\\\"");
+    }
   }
 
   editIf->insertText(line, col, text);
@@ -2196,6 +2211,29 @@
 //  kdDebug(24000) << "Document " << url() << " saved." << endl;
 }
 
+bool Document::saveAs(const KURL& url)
+{
+  bool result = m_doc->saveAs(url);
+  if (result)
+  {
+    m_md5sum = "";
+    if (url.isLocalFile())
+    {
+      QFile f(url.path());
+      if (f.open(IO_ReadOnly))
+      {
+        const char* c = "";
+        KMD5 context(c);
+        context.reset();
+        context.update(f);
+        m_md5sum = context.hexDigest();
+        f.close();
+      }
+    }
+  }
+  return result;
+}
+
 void Document::enableGroupsForDTEP(const QString& dtepName, bool enable)
 {
   if (m_groupsForDTEPs.isEmpty())
Index: quanta/src/quanta.desktop
===================================================================
--- quanta/src/quanta.desktop	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/src/quanta.desktop	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -6,7 +6,6 @@
 Name[hi]=क्वांटा प्लस
 Name[sv]=Quanta plus
 Name[ta]=குவாண்டா ப்ளஸ்
-Name[xx]=xxQuanta Plusxx
 Exec=quanta %U
 Icon=quanta
 Type=Application
@@ -30,6 +29,7 @@
 GenericName[is]=Vefþróunarumhverfi
 GenericName[it]=Ambiente per lo sviluppo Web
 GenericName[ja]=Web 開発環境
+GenericName[lt]=Žiniatinklio kūrimo aplinka
 GenericName[ms]=Persekitaran Pembangunan Web
 GenericName[nl]=Omgeving voor webdevelopment
 GenericName[pl]=Środowisko do tworzenia stron WWW
@@ -44,7 +44,6 @@
 GenericName[tg]=Фазои тайёр кардани-Web
 GenericName[tr]=Web Geliştirme Ortamı
 GenericName[uk]=Середовище веб-розробки
-GenericName[xx]=xxWeb Development Environmentxx
 GenericName[zh_CN]=Web 开发环境
 Comment=Web Development Environment
 Comment[bg]=Среда за разработка на уеб приложения
@@ -65,6 +64,7 @@
 Comment[is]=Vefþróunarumhverfi
 Comment[it]=Ambiente per lo sviluppo Web
 Comment[ja]=Web 開発環境
+Comment[lt]=Žiniatinklio kūrimo aplinka
 Comment[ms]=Persekitaran Pembangunan Web
 Comment[nl]=Omgeving voor webdevelopment
 Comment[pl]=Środowisko do tworzenia stron WWW
@@ -79,6 +79,5 @@
 Comment[tg]=Фазои тайёр кардани-Web
 Comment[tr]=Web Geliştirme Ortamı
 Comment[uk]=Середовище веб-розробки
-Comment[xx]=xxWeb Development Environmentxx
 Comment[zh_CN]=Web 开发环境
 Categories=Qt;KDE;Development;WebDevelopment;
Index: quanta/src/document.h
===================================================================
--- quanta/src/document.h	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/src/document.h	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -147,6 +147,8 @@
   void checkDirtyStatus();
   /** Save the document and reset the dirty status. */
   void save();
+  /** Save the document under a new name and calculate the new md5sum. */
+  bool saveAs(const KURL& url);
   /** Enable or disable the visibility of groups for a DTEP.*/
   void enableGroupsForDTEP(const QString& dtepName, bool enable = true);
   /** Clears the selected DTEP list */
Index: quanta/src/quantaview.cpp
===================================================================
--- quanta/src/quantaview.cpp	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/src/quantaview.cpp	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -1133,7 +1133,7 @@
   }
   if (url.isLocalFile())
   {
-    if (!m_document->doc()->saveAs(url))
+    if (!m_document->saveAs(url))
     {
       fileWatcher->addFile(oldURL.path());
 //      kdDebug(24000) << "addFile[saveDocument]: " << oldURL.path() << endl;
@@ -1151,7 +1151,7 @@
     m_eventLoopStarted = false;
     connect(doc, SIGNAL(canceled(const QString &)), this, SLOT(slotSavingFailed(const QString &)));
     connect(doc, SIGNAL(completed()), this, SLOT(slotSavingCompleted()));
-    m_saveResult = doc->saveAs(url);
+    m_saveResult = m_document->saveAs(url);
     if (m_saveResult)
     {
       //start an event loop and wait until the saving finished
Index: quanta/src/x-webprj.desktop
===================================================================
--- quanta/src/x-webprj.desktop	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/src/x-webprj.desktop	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -17,6 +17,7 @@
 Comment[is]=Quanta verkefnið
 Comment[it]=Progetto Quanta
 Comment[ja]=Quanta プロジェクト
+Comment[lt]=Quanta projektas
 Comment[ms]=Projek Quanta
 Comment[nl]=Quanta project
 Comment[pl]=Projekt Quanta
@@ -32,7 +33,6 @@
 Comment[tg]=Нақшаи Quanta
 Comment[tr]=Quanta Projesi
 Comment[uk]=Проект Quanta
-Comment[xx]=xxQuanta Projectxx
 Comment[zh_CN]=Quanta 工程
 Icon=quanta
 Type=MimeType
Index: quanta/src/quanta_be.desktop
===================================================================
--- quanta/src/quanta_be.desktop	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/src/quanta_be.desktop	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -4,7 +4,6 @@
 Name[br]=Quanta+ BE
 Name[sv]=Quanta plus BE
 Name[ta]=குவாண்டா ப்ளஸ் BE
-Name[xx]=xxQuanta Plus BExx
 GenericName=Web Development Environment
 GenericName[bg]=Разработка на уеб приложения
 GenericName[ca]=Entorn de desenvolupament web
@@ -23,6 +22,7 @@
 GenericName[is]=Vefþróunarumhverfi
 GenericName[it]=Ambiente per lo sviluppo Web
 GenericName[ja]=Web 開発環境
+GenericName[lt]=Žiniatinklio kūrimo aplinka
 GenericName[ms]=Persekitaran Pembangunan Web
 GenericName[nl]=Omgeving voor webdevelopment
 GenericName[pl]=Środowisko do tworzenia stron WWW
@@ -37,7 +37,6 @@
 GenericName[tg]=Фазои тайёр кардани-Web
 GenericName[tr]=Web Geliştirme Ortamı
 GenericName[uk]=Середовище веб-розробки
-GenericName[xx]=xxWeb Development Environmentxx
 GenericName[zh_CN]=Web 开发环境
 Exec=quanta_be
 Icon=quanta_be
@@ -63,6 +62,7 @@
 Comment[is]=Vefþróunarumhverfi
 Comment[it]=Ambiente per lo sviluppo Web
 Comment[ja]=Web 開発環境
+Comment[lt]=Žiniatinklio kūrimo aplinka
 Comment[ms]=Persekitaran Pembangunan Web
 Comment[nl]=Omgeving voor webdevelopment
 Comment[pl]=Środowisko do tworzenia stron WWW
@@ -77,6 +77,5 @@
 Comment[tg]=Фазои тайёр кардани-Web
 Comment[tr]=Web Geliştirme Ortamı
 Comment[uk]=Середовище веб-розробки
-Comment[xx]=xxWeb Development Environmentxx
 Comment[zh_CN]=Web 开发环境
 Categories=Qt;KDE;Development;
Index: quanta/src/quanta.cpp
===================================================================
--- quanta/src/quanta.cpp	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/src/quanta.cpp	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -638,11 +638,6 @@
 //TODO: Implement it!
 }
 
-void QuantaApp::slotFileClose()
-{
-  ViewManager::ref()->removeActiveView();
-}
-
 void QuantaApp::slotFileClose(const KURL &url)
 {
   QuantaView *view = ViewManager::ref()->isOpened(url);
@@ -870,7 +865,10 @@
           imgTag += QuantaCommon::quoteAttributeValue(height);
           imgTag += QuantaCommon::attrCase(" border=");
           imgTag += QuantaCommon::quoteAttributeValue(QString("%1").arg(0));
-          imgTag += ">";
+          if (w->currentDTD(true)->singleTagStyle == "xml")
+            imgTag += " />";
+          else
+            imgTag += ">";
           w->insertTag(imgTag);
           isImage = true;
         }
@@ -2998,6 +2996,43 @@
   dlg.exec();
 }
 
+void QuantaApp::setDtep(const QString& dtepName, bool convert)
+{
+  Document *w = ViewManager::ref()->activeDocument();
+  if (w)
+  {
+    QString dtep = DTDs::ref()->getDTDNameFromNickName(dtepName);
+    if (!DTDs::ref()->find(dtep))
+      return;
+    w->setDTDIdentifier(dtep);
+    const DTDStruct *dtd = DTDs::ref()->find(w->getDTDIdentifier());
+    if (convert && dtd->family == Xml)
+    {
+      Tag *tag = 0L;
+      w->findDTDName(&tag);
+      if (tag)
+      {
+        int bLine, bCol, eLine, eCol;
+        tag->beginPos(bLine,bCol);
+        tag->endPos(eLine,eCol);
+        w->editIf->removeText(bLine, bCol, eLine, eCol+1);
+        w->viewCursorIf->setCursorPositionReal((uint)bLine, (uint)bCol);
+        w->insertText("<!DOCTYPE" + dtd->doctypeStr +">");
+        delete tag;
+      } else
+      {
+        w->viewCursorIf->setCursorPositionReal(0,0);
+        w->insertText("<!DOCTYPE" + dtd->doctypeStr + ">\n");
+      }
+    }
+    slotLoadToolbarForDTD(w->getDTDIdentifier());
+    QuantaView *view = ViewManager::ref()->activeView();
+    if (view)
+      view->activated();
+    reparse(true);
+  }
+}
+
 void QuantaApp::slotChangeDTD()
 {
   Document *w = ViewManager::ref()->activeDocument();
@@ -3010,8 +3045,6 @@
     int pos = -1;
     int defaultIndex = 0;
 
-    Tag *tag = 0L;
-    w->findDTDName(&tag);
     QString oldDtdName = w->getDTDIdentifier();
     QString defaultDocType = Project::ref()->defaultDTD();
     QStringList lst = DTDs::ref()->nickNameList(true);
@@ -3036,32 +3069,8 @@
     dtdWidget->adjustSize();
     if (dlg.exec())
     {
-      w->setDTDIdentifier(DTDs::ref()->getDTDNameFromNickName(dtdWidget->dtdCombo->currentText()));
-      const DTDStruct *dtd = DTDs::ref()->find(w->getDTDIdentifier());
-      if (dtdWidget->convertDTD->isChecked() && dtd->family == Xml)
-      {
-        if (tag)
-        {
-          int bLine, bCol, eLine, eCol;
-          tag->beginPos(bLine,bCol);
-          tag->endPos(eLine,eCol);
-          w->editIf->removeText(bLine, bCol, eLine, eCol+1);
-          w->viewCursorIf->setCursorPositionReal((uint)bLine, (uint)bCol);
-          w->insertText("<!DOCTYPE" + dtd->doctypeStr +">");
-          delete tag;
-        } else
-        {
-          w->viewCursorIf->setCursorPositionReal(0,0);
-          w->insertText("<!DOCTYPE" + dtd->doctypeStr + ">\n");
-        }
-      }
+      setDtep(dtdWidget->dtdCombo->currentText(), dtdWidget->convertDTD->isChecked());
     }
-
-    slotLoadToolbarForDTD(w->getDTDIdentifier());
-    QuantaView *view = ViewManager::ref()->activeView();
-    if (view)
-      view->activated();
-    reparse(true);
   }
 }
 
Index: quanta/src/quantaview.h
===================================================================
--- quanta/src/quantaview.h	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/src/quantaview.h	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -19,6 +19,7 @@
 #define QUANTAVIEW_H
 
 // include files for Qt
+#include <qguardedptr.h>
 #include <qwidget.h>
 #include <qptrlist.h>
 #include <qvaluelist.h>
@@ -214,7 +215,7 @@
   Document *m_document;
   QuantaPlugin *m_plugin;
   QWidget *m_customWidget; ///<view holds a custom widget, eg. a documentation
-  KafkaDocument *m_kafkaDocument;
+  QGuardedPtr<KafkaDocument> m_kafkaDocument;
   QSplitter *m_splitter;
   QGridLayout *m_viewLayout;
   int m_currentViewsLayout; ///< holds the current layout, which can be SourceOnly, VPLOnly or SourceAndVPL
Index: quanta/src/dcopwindowmanagerif.h
===================================================================
--- quanta/src/dcopwindowmanagerif.h	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/src/dcopwindowmanagerif.h	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -34,6 +34,7 @@
   virtual void newDebuggerPosition(const QString &file, int lineNumber) = 0;
   virtual void openFile(const QString &file, int lineNumber, int columnNumber) = 0;
   virtual QString saveCurrentFile() = 0;
+  virtual void setDtep(const QString& dtepName, bool convert) = 0;
   virtual QString documentFolderForURL(const QString &url) = 0;
   virtual QString urlWithPreviewPrefix(const QString &url) = 0;
   virtual void addFileToProject(const QString &url) = 0;
Index: quanta/parts/kafka/htmldocumentproperties.cpp
===================================================================
--- quanta/parts/kafka/htmldocumentproperties.cpp	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/parts/kafka/htmldocumentproperties.cpp	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -541,7 +541,7 @@
 		finalURL = url.url();
 		if (url.protocol() == view->document()->url().protocol())
 			finalURL.remove(0, url.protocol().length() + 1);
-		if(linkNode->tag->attributeValue("rel").lower() != "stylesheet");
+		if(linkNode->tag->attributeValue("rel").lower() != "stylesheet")
 			linkNode->tag->editAttribute("rel", "stylesheet");
 		linkNode->tag->editAttribute("href", finalURL);
 		linkNode->tag->setCleanStrBuilt(false);
Index: quanta/parts/kafka/wkafkapart.cpp
===================================================================
--- quanta/parts/kafka/wkafkapart.cpp	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/parts/kafka/wkafkapart.cpp	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -1081,7 +1081,12 @@
                     text = ">";
 		}
 		else
-			text = "<" + QuantaCommon::tagCase(node->tag->name) + ">";
+    {
+      if (node->tag->tagStr() == "-->")
+        text = "-->";
+      else
+			 text = "<" + QuantaCommon::tagCase(node->tag->name) + ">";
+    }
 		bCol += text.length();
 		eCol = bCol - 1;
 		eLine = bLine;
Index: quanta/utility/tagaction.cpp
===================================================================
--- quanta/utility/tagaction.cpp	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/utility/tagaction.cpp	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -1145,7 +1145,7 @@
     long cursor_offset = selection.cursorOffset();
     kafkaCommon::splitStartAndEndNodeSubtree(start_node, start_offset, end_node, end_offset, commonParent, 
                                              commonParentStartChildLocation, commonParentEndChildLocation,
-                                             selection, false, modifs);
+                                             selection, 0, modifs);
 
     Q_ASSERT(start_node != end_node);
     
@@ -1221,7 +1221,7 @@
     int cursor_offset = selection.cursorOffset();*/
     kafkaCommon::splitStartAndEndNodeSubtree(start_node, start_offset, end_node, end_offset, commonParent, 
                                              commonParentStartChildLocation, commonParentEndChildLocation,
-                                             selection, /*cursor_node, cursor_offset, */false, modifs); 
+                                             selection, /*cursor_node, cursor_offset, */0, modifs); 
     
 //     kafkaCommon::coutTree(baseNode, 3);
     
Index: quanta/project/project.cpp
===================================================================
--- quanta/project/project.cpp	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/project/project.cpp	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -240,7 +240,7 @@
 
 void Project::slotOpenProject(const KURL &url)
 {
-    kdDebug(24000) << "Open recent project: " << url <<endl;
+  kdDebug(24000) << "Open recent project: " << url <<endl;
   if ( !url.isEmpty() )
   {
     if ( !QExtFileInfo::exists(url, d->m_mainWindow) )
Index: quanta/project/uploadprofiledlgs.ui
===================================================================
--- quanta/project/uploadprofiledlgs.ui	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/project/uploadprofiledlgs.ui	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -235,7 +235,7 @@
                 <bool>false</bool>
             </property>
             <property name="toolTip" stdset="0">
-                <string>Keep the password in a file. DANGEROUS! Read the What's This text.</string>
+                <string>Keep the password in a file. DANGEROUS! Read the "What's This?" text.</string>
             </property>
             <property name="whatsThis" stdset="0">
                 <string>&lt;qt&gt;If you check this box, the password will be saved in the local Quanta configuration file, so every time you start Quanta and load this project, the password will be loaded and you don't have to enter it. Altough the local configuration file is readable only by you and the password
Index: quanta/project/teammembersdlg.cpp
===================================================================
--- quanta/project/teammembersdlg.cpp	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/project/teammembersdlg.cpp	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -76,7 +76,7 @@
         } else
         if (nickName.isEmpty())
         {
-           KMessageBox::error(this, i18n("The nickname cannot be empty as it is used as an unique identifier."));
+           KMessageBox::error(this, i18n("The nickname cannot be empty as it is used as a unique identifier."));
            addDlg.show();
         } else
         if (!checkDuplicates(0L, name, nickName, email, role, task, subProject))
@@ -146,7 +146,7 @@
         } else
          if (nickName.isEmpty())
         {
-           KMessageBox::error(this, i18n("The nickname cannot be empty as it is used as an unique identifier."));
+           KMessageBox::error(this, i18n("The nickname cannot be empty as it is used as a unique identifier."));
            editDlg.show();
         } else
        if (!checkDuplicates(item, name, nickName, email, role, task, subProject))
Index: quanta/data/config/quanta_preview_config.desktop
===================================================================
--- quanta/data/config/quanta_preview_config.desktop	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/data/config/quanta_preview_config.desktop	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -8,7 +8,7 @@
 Name[ca]=Configura la part del fullejador KHTML
 Name[cs]=Nastavení KHTML Part
 Name[da]=Indstil KHTML browser-part
-Name[de]=Einrichten der  KHTML-Browserkomponente
+Name[de]=Einrichten der KHTML-Browserkomponente
 Name[el]=Ρύθμιση του KHTML περιηγητή
 Name[es]=Configurara la parte del navegador KHTML
 Name[et]=KHTML brauseri komponendi seadistamine
@@ -19,6 +19,7 @@
 Name[is]=Stilla KHTML vafrahluti
 Name[it]=Parte browser configura KHTML
 Name[ja]=KHTMLブラウザ部を設定
+Name[lt]=Konfigūruoti KHTML naršyklės dalį
 Name[ms]=Selaraskan Bahagian Pelayar KHTML
 Name[nl]=KHTML Browser Part configuratie
 Name[pl]=Konfiguracja osadzalnej części przeglądarki KHTML
@@ -33,5 +34,4 @@
 Name[tg]=Танзими қисми браузери KHTML
 Name[tr]=KHTML Tarayıcısı Parçasını Yapılandır
 Name[uk]=Налаштувати компонент перегляду KHTML
-Name[xx]=xxConfigure KHTML Browser Partxx
 Name[zh_CN]=配置 KHTML 浏览器部件
Index: quanta/data/dtep/xhtml-basic/description.rc
===================================================================
--- quanta/data/dtep/xhtml-basic/description.rc	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/data/dtep/xhtml-basic/description.rc	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -1,6 +1,6 @@
 [General]
 Name = -//W3C//DTD XHTML Basic 1.0//EN
-NickName = XHTML Basic 1.0
+NickName = XHTML 1.0 Basic
 URL = http://www.w3.org/TR/xhtml-basic/xhtml-basic10.dtd
 DoctypeString = html PUBLIC "-//W3C//DTD XHTML Basic 1.0//EN" "http://www.w3.org/TR/xhtml-basic/xhtml-basic10.dtd"
 DefaultExtension = html
Index: quanta/treeviews/structtreeview.cpp
===================================================================
--- quanta/treeviews/structtreeview.cpp	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/treeviews/structtreeview.cpp	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -776,7 +776,7 @@
         m_marker = current_item;
             
     if(current_item == m_draggedItem ||
-       !(e->provides("text/x-struct_tree_tag_item") || e->source() == this) || !m_marker)
+       !(e->provides("text/x-struct_tree_tag_item") || e->source() == (QWidget *)this) || !m_marker)
     {
         e->accept(false);
         last_accept = false;
Index: quanta/treeviews/basetreeview.cpp
===================================================================
--- quanta/treeviews/basetreeview.cpp	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/treeviews/basetreeview.cpp	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -1212,7 +1212,7 @@
     KTempFile *tempFile = new KTempFile(tmpDir);
     tempFile->setAutoDelete(true);
     tempFile->close();
-    if (QuantaNetAccess::copy(KURL::fromPathOrURL(tempFile->name()), url, this));
+    if (QuantaNetAccess::copy(KURL::fromPathOrURL(tempFile->name()), url, this))
     {
       emit openFile(url);
     }
Index: quanta/treeviews/projecttreeview.cpp
===================================================================
--- quanta/treeviews/projecttreeview.cpp	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ quanta/treeviews/projecttreeview.cpp	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -405,7 +405,7 @@
     KTempFile *tempFile = new KTempFile(tmpDir);
     tempFile->setAutoDelete(true);
     tempFile->close();
-    if (QExtFileInfo::copy(KURL::fromPathOrURL(tempFile->name()), url));
+    if (QExtFileInfo::copy(KURL::fromPathOrURL(tempFile->name()), url))
     {
       emit insertToProject(url);
       emit openFile(url);
Index: kxsldbg/kxsldbgpart/kxsldbg_part.desktop
===================================================================
--- kxsldbg/kxsldbgpart/kxsldbg_part.desktop	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ kxsldbg/kxsldbgpart/kxsldbg_part.desktop	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -9,7 +9,6 @@
 Name[sv]=Kxsldbg-delprogram
 Name[ta]=KXsldbg பகுதி
 Name[tg]=Қисми KXsldbg
-Name[xx]=xxKXsldbgPartxx
 MimeType=text/english;text/plain;text/x-makefile;text/x-c++hdr;text/x-c++src;text/x-chdr;text/x-csrc;text/x-java;text/x-moc;text/x-pascal;text/x-tcl;text/x-tex;application/x-shellscript;text/x-c;text/x-c++;
 ServiceTypes=KParts/ReadOnlyPart
 X-KDE-Library=libkxsldbgpart
Index: kxsldbg/kxsldbg.desktop
===================================================================
--- kxsldbg/kxsldbg.desktop	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ kxsldbg/kxsldbg.desktop	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -3,7 +3,6 @@
 Name=KXSLDbg
 Name[sv]=Kxsldbg
 Name[ta]=KXSLDபிஜி
-Name[xx]=xxKXSLDbgxx
 GenericName=XSLT Debugger
 GenericName[br]=Dizraener XSLT
 GenericName[ca]=Depurador XSLT
@@ -13,7 +12,7 @@
 GenericName[el]=Αποσφαλματωτής XSLT
 GenericName[es]=Depurador XSLT
 GenericName[et]=XSLT silur
-GenericName[eu]=XSLT araztatzailea
+GenericName[eu]=XSLT araztailea
 GenericName[fi]=XSLT debuggeri
 GenericName[fr]=Débogueur XSLT
 GenericName[ga]=Dífhabhtóir XSLT
@@ -23,6 +22,7 @@
 GenericName[is]=XSLT aflúsari
 GenericName[it]=Debugger per XSLT
 GenericName[ja]=XSLT デバッガ
+GenericName[lt]=XSLT derintuvė
 GenericName[ms]=Penyah-ralat XSLT
 GenericName[nl]=XSLT-debugger
 GenericName[pl]=Debuger XSLT
@@ -38,7 +38,6 @@
 GenericName[tg]=Ғалатёби XSLT
 GenericName[tr]=XSLT Hata Ayıklayıcısı
 GenericName[uk]=Зневаджувач XSLT
-GenericName[xx]=xxXSLT Debuggerxx
 GenericName[zh_CN]=XSLT 调试器
 Exec=kxsldbg %i %m -caption "%c"
 Icon=kxsldbg
Index: kimagemapeditor/kimagemapeditorpart.desktop
===================================================================
--- kimagemapeditor/kimagemapeditorpart.desktop	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ kimagemapeditor/kimagemapeditorpart.desktop	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -21,6 +21,7 @@
 Comment[is]=HTML myndakortsritill
 Comment[it]=Un editor per le mappe di immagini HTML
 Comment[ja]=HTMLイメージマップエディター
+Comment[lt]=HTML paveikslėlių žemėlapių rengyklė
 Comment[ms]=Penyunting Peta Imej HTML
 Comment[nl]=Een hulpmiddel om HTML imagemaps te maken
 Comment[pl]=Edytor map obrazków HTML
@@ -35,7 +36,6 @@
 Comment[tg]=Муҳаррири тасвири ҳаритаи HTML
 Comment[tr]=Bir HTML resim_haritası düzenleyicisi
 Comment[uk]=Редактор imagemap HTML
-Comment[xx]=xxAn HTML imagemap editorxx
 Comment[zh_CN]=HTML 图像映射编辑器
 Name=KImageMapEditor
 Name[pt_BR]=KEditor de Mapeamento de Imagem
@@ -43,4 +43,3 @@
 Name[ta]=Kவடிவம் வரைப்படம் திருத்தி
 Name[tg]=KМуҳаррири тасвири ҳарита
 Name[tr]=K Resim Haritası Düzenleyicisi
-Name[xx]=xxKImageMapEditorxx
Index: kimagemapeditor/kimagemapeditor.desktop
===================================================================
--- kimagemapeditor/kimagemapeditor.desktop	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ kimagemapeditor/kimagemapeditor.desktop	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -11,7 +11,6 @@
 Name[ta]=Kவடிவம் வரைப்படம் திருத்தி
 Name[tg]=KМуҳаррири тасвири ҳарита
 Name[tr]=K Resim Haritası Düzenleyicisi
-Name[xx]=xxKImageMapEditorxx
 GenericName=HTML Image Map Editor
 GenericName[ca]=Un editor de mapes d'imatge HTML
 GenericName[cs]=Editor HTML map
@@ -27,6 +26,7 @@
 GenericName[is]=HTML myndakortsritill
 GenericName[it]=Editor per le mappe di immagini HTML
 GenericName[ja]=HTMLイメージマップエディター
+GenericName[lt]=HTML paveikslėlių žemėlapių rengyklė
 GenericName[ms]=Penyunting Peta Imej HTML
 GenericName[nl]=HTML imagemap editor
 GenericName[pl]=Edytor map obrazków HTML
@@ -40,6 +40,5 @@
 GenericName[tg]=Муҳаррири тасвири ҳаритаи HTML
 GenericName[tr]=HTML Resim Haritası Düzenleyicisi
 GenericName[uk]=Редактор карт зображень HTML
-GenericName[xx]=xxHTML Image Map Editorxx
 GenericName[zh_CN]=HTML 图像映射编辑器
 Categories=Qt;KDE;Development;WebDevelopment;
Index: klinkstatus/src/klinkstatus.desktop
===================================================================
--- klinkstatus/src/klinkstatus.desktop	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ klinkstatus/src/klinkstatus.desktop	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -4,7 +4,6 @@
 Name[sv]=Klinkstatus
 Name[ta]=Kதொகுதி நிலைமை
 Name[tr]=K Köprü Durumu
-Name[xx]=xxKLinkStatusxx
 Exec=klinkstatus %i %m -caption "%c"
 Icon=klinkstatus
 Type=Application
@@ -26,6 +25,7 @@
 GenericName[is]=Yfirfer tengla
 GenericName[it]=Controllo dei collegamenti
 GenericName[ja]=リンクチェッカー
+GenericName[lt]=Nuorody tikrintuvė
 GenericName[ms]=Pemeriksa Pautan
 GenericName[nl]=Linkchecker
 GenericName[pl]=Program sprawdzający odnośniki
@@ -40,7 +40,6 @@
 GenericName[tg]=Тафтиши истинод
 GenericName[tr]=Bağlantı Kontrolcüsü
 GenericName[uk]=Перевірка посилань
-GenericName[xx]=xxLink Checkerxx
 GenericName[zh_CN]=链接检查器
 Categories=Qt;KDE;Development;WebDevelopment;
 
Index: klinkstatus/src/klinkstatus_part.desktop
===================================================================
--- klinkstatus/src/klinkstatus_part.desktop	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ klinkstatus/src/klinkstatus_part.desktop	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -8,7 +8,6 @@
 Name[sv]=Klinkstatus-delprogram
 Name[ta]=Kதொகுதி நிலைமை பகுதி
 Name[tg]=Қисми KLinkStatus
-Name[xx]=xxKLinkStatusPartxx
 Icon=klinkstatus
 MimeType=text/english;text/plain;text/x-makefile;text/x-c++hdr;text/x-c++src;text/x-chdr;text/x-csrc;text/x-java;text/x-moc;text/x-pascal;text/x-tcl;text/x-tex;application/x-shellscript;text/x-c;text/x-c++;
 ServiceTypes=KParts/ReadOnlyPart,KParts/ReadWritePart
Index: kommander/editor/kmdr-editor.desktop
===================================================================
--- kommander/editor/kmdr-editor.desktop	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ kommander/editor/kmdr-editor.desktop	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -19,6 +19,7 @@
 GenericName[is]=Dynamic Dialog ritillinn
 GenericName[it]=Editor per le maschere dinamiche
 GenericName[ja]=動的ダイアログエディター
+GenericName[lt]=Dinaminė dialogų rengyklė
 GenericName[ms]=Penyunting Dialog Dinamik
 GenericName[nl]=Dynamische dialoog-editor
 GenericName[pl]=Edytor do dynamicznych okien dialogowych
@@ -33,7 +34,6 @@
 GenericName[tg]=Муҳаррири диалоги динамика
 GenericName[tr]=Dinamik Diyalog Düzenleyicisi
 GenericName[uk]=Динамічний редактор вікон
-GenericName[xx]=xxDynamic Dialog Editorxx
 GenericName[zh_CN]=动态对话框编辑器
 Name=Kommander Editor
 Name[br]=Aozer Kommander
@@ -54,6 +54,7 @@
 Name[is]=Kommander ritillinn
 Name[it]=Editor Kommander
 Name[ja]=Kommander エディター
+Name[lt]=Kommander rengyklė
 Name[ms]=Penyunting Kommander
 Name[pa]=Kommander ਸੰਪਾਦਕ
 Name[pl]=Edytor Kommander
@@ -68,7 +69,6 @@
 Name[tg]=Муҳаррири Kommander
 Name[tr]=Kommander Düzenleyici
 Name[uk]=Редактор Kommander
-Name[xx]=xxKommander Editorxx
 Name[zh_CN]=Kommander 编辑器
 Exec=kmdr-editor %F
 Icon=kommander
@@ -93,6 +93,7 @@
 Comment[is]=Dynamic Dialog ritillinn
 Comment[it]=Editor per le maschere dinamiche
 Comment[ja]=動的ダイアログエディター
+Comment[lt]=Dinaminė dialogų rengyklė
 Comment[ms]=Penyunting Dialog Dinamik
 Comment[nl]=Dynamische dialoog-editor
 Comment[pl]=Edytor do dynamicznych okien dialogowych
@@ -107,7 +108,6 @@
 Comment[tg]=Муҳаррири диалоги динамика
 Comment[tr]=Dinamik Diyalog Düzenleyicisi
 Comment[uk]=Динамічний редактор вікон
-Comment[xx]=xxDynamic Dialog Editorxx
 Comment[zh_CN]=动态对话框编辑器
 MimeType=application/x-kommander
 DocPath=kommander/index.html
Index: kommander/x-kommander.desktop
===================================================================
--- kommander/x-kommander.desktop	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ kommander/x-kommander.desktop	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -20,6 +20,7 @@
 Comment[is]=Kommander skrá
 Comment[it]=File di Kommander
 Comment[ja]=Kommander ファイル
+Comment[lt]=Kommander byla
 Comment[ms]=Fail Kommander
 Comment[nl]=Kommander-dialoogbestand
 Comment[pa]=Kommander ਫਾਇਲ
@@ -35,7 +36,6 @@
 Comment[tg]=Файли Kommander
 Comment[tr]=Kommander Dosyası
 Comment[uk]=Файл Kommander
-Comment[xx]=xxKommander Filexx
 Comment[zh_CN]=Kommander 文件
 Icon=widget_doc
 Type=MimeType
Index: kommander/executor/kmdr-executor.desktop
===================================================================
--- kommander/executor/kmdr-executor.desktop	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ kommander/executor/kmdr-executor.desktop	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -24,6 +24,7 @@
 Comment[is]=Executor fyrir Kommander glugga
 Comment[it]=Esecutore per maschere di Kommander
 Comment[ja]=Kommander ダイアログ用の Executor
+Comment[lt]=Kommander dialogų vykdyklė
 Comment[ms]=Pelaksana untuk dialog Kommander
 Comment[nl]=Kommander-dialogen uitvoeren
 Comment[pl]=Uruchamianie z okien dialogowych Kommandera
@@ -38,7 +39,6 @@
 Comment[tg]=Иҷрокунанда барои диалогҳои Kommander
 Comment[tr]=Kommander diyalogları için çalıştırıcı
 Comment[uk]=Запускач для вікон Kommander
-Comment[xx]=xxExecutor for Kommander dialogsxx
 Comment[zh_CN]=Kommander 对话框执行器
 Terminal=false
 Name=Kommander Executor
@@ -54,6 +54,7 @@
 Name[gl]=Executor de Kommander
 Name[hi]=कमांडर कार्यपालक
 Name[it]=Esecutore per Kommander
+Name[lt]=Kommander vykdyklė
 Name[ms]=Pelaksana Kommander
 Name[nl]=Kommander Uitvoerder
 Name[pl]=Kommander: uruchamianie
@@ -67,6 +68,5 @@
 Name[tg]=Иҷрокунандаи Kommander
 Name[tr]=Kommander Çalıştırıcısı
 Name[uk]=Запускач Kommander
-Name[xx]=xxKommander Executorxx
 Name[zh_CN]=Kommander 执行器
 MimeType=application/x-kommander
Index: kommander/widgets/widgets.desktop
===================================================================
--- kommander/widgets/widgets.desktop	(.../tags/KDE/3.5.0/kdewebdev)	(revision 492851)
+++ kommander/widgets/widgets.desktop	(.../branches/KDE/3.5/kdewebdev)	(revision 492851)
@@ -6,6 +6,7 @@
 Name[cs]=Widgety
 Name[cy]=Celfigion
 Name[da]=Kontroller
+Name[el]=Γραφικά συστατικά
 Name[et]=Elemendid
 Name[fa]=تکه‌ها
 Name[fi]=Elementit
@@ -14,6 +15,7 @@
 Name[hi]=विजेट्स
 Name[hu]=Grafikai elemek
 Name[ja]=ウィジェット
+Name[lt]=Valdikliai
 Name[ms]=Wijet
 Name[pl]=Elementy interfejsu
 Name[pt]=Elementos
@@ -27,7 +29,6 @@
 Name[tg]=Васеъкуниҳо
 Name[tr]=Bileşenler
 Name[uk]=Віджети
-Name[xx]=xxWidgetsxx
 Name[zh_CN]=部件
 Exec=widgets %i %m -caption "%c"
 Type=Application

Property changes on: .
___________________________________________________________________
Name: svn:externals
   + admin https://svn.kde.org/home/kde/branches/KDE/3.5/kde-common/admin


